### PR TITLE
Speech

### PR DESCRIPTION
**Pull request by jbrennan**
Now your prototopes can really say something!

### CODE REVIEW

File: PrototopeJSBridge/SpeechBridge.swift [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+		context.setObject(self, forKeyedSubscript: "Speech")
+		let speechBridge = context.objectForKeyedSubscript("Speech")
+		
+		speechBridge.setFunctionForKey("say", fn: sayTrampoline)
+	}
+	
+}
+
+
+let sayTrampoline: @objc_block JSValue -> Void = { args in
```
Comment (NachoSoto) [inline PrototopeJSBridge/SpeechBridge.swift]: Whoa, `@objc_block`? TIL!

Comment (saniul) [inline PrototopeJSBridge/SpeechBridge.swift]: @NachoSoto Also worth mentioning – https://github.com/Khan/Prototope/pull/20#discussion_r24793735

Comment (jbrennan) [inline PrototopeJSBridge/SpeechBridge.swift]: Yeah, TIL too! We use it elsewhere in prototope for animation bridging I think.

**Comment by NachoSoto** [comment]
Not a fan of types with all static methods, but it's probably the most convenient API for this :)

**Comment by jbrennan** [comment]
@NachoSoto Yeah, I went back and forth on that. I settled on static methods because it really only makes sense to speak one thing at a time, and this lets you queue things up without having to figure out when to make the next thing speak.

**Comment by NachoSoto** [comment]
Yup! :shipit: 
Should I merge? since you assigned it to me

**Comment by jbrennan** [comment]
@NachoSoto Yep!

**Pull request merged**